fixtures/eurolite/tmh-w400.json

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "TMH-W400",
+  "shortName": "tmh400",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["McMars"],
+    "createDate": "2023-10-13",
+    "lastModifyDate": "2023-10-13"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/de/mpn51785930-eurolite-led-tmh-w400-moving-head-wash-zoom.html"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/de/mpn51785930-eurolite-led-tmh-w400-moving-head-wash-zoom.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=6aR_Uofh7ms&ab_channel=eurolitevideo"
+    ]
+  },
+  "physical": {
+    "dimensions": [300, 230, 450],
+    "weight": 10,
+    "power": 400,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 4000
+    },
+    "lens": {
+      "degreesMinMax": [9, 45]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "capability": {
+        "type": "Pan",
+        "angle": "540deg"
```

Comment:
Presumably the pan/tilt channels should have "angleStart"/"angleEnd" rather than just "angle".